doc/changes/names.inc

```diff
@@ -22,7 +22,7 @@
 
 .. _Andrew Dykstra: https://github.com/adykstra
 
-.. _Romain Trachel: http://www.lscp.net/braware/trachelBr.html
+.. _Romain Trachel: https://fr.linkedin.com/in/trachelr
```

Comment:
```suggestion
.. _Romain Trachel: https://github.com/trachelr
```

Reply:
I used the URL in his GitHub profile

Reply:
(to me if someone has it set, it's the URL they want you looking at to get information on them)